cmd/virt-chroot/main_test.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Virt chroot sanity", func() {
+	It("virt-chroot should not be bloated with dependencies", func() {
+		path, err := os.Executable()
```

Comment:
Technically speaking, this checks the size of the test binary. Should not matter though.

Reply:
yeah not perfect but should provide some defence against obvious errors like this simple binary growing by 5x